src/reducers/cursors.js

```diff
@@ -0,0 +1,27 @@
+import { Map } from 'immutable';
+import {
+  ENTRIES_SUCCESS,
+} from 'Actions/entries';
+
+// Since pagination can be used for a variety of views (collections
+// and searches are the most common examples), we namespace cursors by
+// their type before storing them in the state.
+export const collectionEntriesCursorKey = collectionName => `collectionEntries.${ collectionName }`;
```

Comment:
This allows us to use pagination only for collections at the moment, while still using a Redux structure that will support other uses of cursors.